### ISSUE TITLE
only use unit description when meaningful

### ISSUE THREAD

**Issue by crotwell** [FDSN/StationXML#53]
Lots of stationxml in the real world and in the examples contain units like:
```
             <InputUnits>
                <Name>M/S**2</Name>
                <Description>Acceleration in Meters Per Second Per Second</Description>
              </InputUnits>
```
```
              <InputUnits>
                <Name>V</Name>
                <Description>Volts</Description>
              </InputUnits>
```
```
            <OutputUnits>
              <Name>COUNTS</Name>
              <Description>Digital Counts</Description>
            </OutputUnits>
```

The use of a description for the commonly used units, especially when it is used over and over within the stationxml file, is verbose and adds nothing to understanding. 

Add sentence to unit [Description](http://docs.fdsn.org/projects/stationxml/en/draft/reference.html#response-stage-poleszeros-inputunits-description) saying: 
> Description is only needed when the unit name is not a well know SI unit or there is need for clarification. Prefer not to use a Description for common units like m/s, m/s**2, count, etc.

Add sentence to [Section 5.4](http://docs.fdsn.org/projects/stationxml/en/draft/appendices.html#unit-naming-rules) saying similar.

**Comment by crotwell** [comment]
Here is an example from the stationxml docs that shows why including a descriptions of units can be bad when the unit is well defined from just the name:

http://docs.fdsn.org/projects/stationxml/en/latest/response.html#ysi-44031-thermistor

```

           <Stage number="4">
              <Coefficients name=" DECIMATION">
              <InputUnits>
                 <Name>counts</Name>
                 <Description>Volts</Description>
              </InputUnits>
```

**Comment by rcasey-iris** [comment]
> Here is an example from the stationxml docs that shows why including a descriptions of units can be bad when the unit is well defined from just the name:
> 
> http://docs.fdsn.org/projects/stationxml/en/latest/response.html#ysi-44031-thermistor
> 
> ```
> 
>            <Stage number="4">
>               <Coefficients name=" DECIMATION">
>               <InputUnits>
>                  <Name>counts</Name>
>                  <Description>Volts</Description>
>               </InputUnits>
> ```

I'm not sure this makes the case for elimination of redundancy.  The above example is just plain incongruent.

If there is to be a mode of compression from redundancy, a top-level definition structure would need to be defined in the document to apply to all sub-elements that use unit descriptions.  StationXML is very hierarchical in nature and doesn't understand the notion of the first sub-element describing a unit, so no further sub-elements need to.  In XML, every sub-element needs to be standalone-complete.

Just not sure that we're saving much here, but that's just my view.

**Comment by crotwell** [comment]
@rcasey-iris I don't understand your comment about "compression".

I am not advocating this to save space, although it would. I just think that having a description for commonly used SI units is at best pointless and at worst adds confusion. 

I am not saying we should prevent description, but rather to prefer not to use description unless it adds information. There is no need to further describe "m/s", the symbol is sufficient.

Am I misunderstanding you?

**Comment by crotwell** [comment]
PR #121 if this change is accepted

**Comment by crotwell** [comment]
fix in #121
